### PR TITLE
feat(dataplane_ut): add passthrough pipeline helper for harness tests

### PR DESCRIPTION
**Pull request by 3Hren**
Add `dataplane_ut_add_passthrough_pipeline`, which wires an empty-chain pipeline to a device so `config_gen_ectx` becomes non-NULL and packets flow end-to-end through the in-process dataplane. This unlocks pipeline-level harness tests and benchmarks that previously could not run a real pipeline round.

Add a unit test exercising the helper: an empty passthrough pipeline forwards all input packets to the output with none dropped.

### CODE REVIEW

File: lib/dataplane_ut/dataplane_ut.c [hunk 1]
```diff
@@ -401,3 +404,148 @@ dataplane_ut_run(
...
+		    yanet_error_message(err));
+		yanet_error_free(err);
+		cp_device_free(cp_device);
+		cp_device_config_fini(&device_cfg);
+		return -1;
+	}
+	cp_device_config_fini(&device_cfg);
+
+	struct cp_device *devices[] = {cp_device};
+	if (agent_update_devices(ut->agent, 1, devices, &err) != 0) {
```
[the remark this answers is not in the document]
Comment (3Hren) [reply lib/dataplane_ut/dataplane_ut.c:536]: Addressed in this revision: the helper now returns the assigned device index via the new `out_tx_device_id` out-parameter (the topology position, which equals the registry slot and the `tx_device_id` that `dataplane_ut_run` routes by), and documents that the caller must use it. A second test case binds the passthrough pipeline to the *second* configured device and asserts both `out_tx_device_id == 1` and full passthrough, which would crash/fail on the previous behavior.
